kili/mutations/asset/helpers.py

```diff
@@ -79,15 +82,14 @@ def process_content(input_type: str,
     Process the array of contents
     """
     if input_type in ['IMAGE', 'PDF']:
-        return [content if is_url(content)
-                else (content
-                      if (json_content_array is not None and json_content_array[i] is not None)
-                      else (encode_base64(content) if check_file_mime_type(content, input_type)
-                            else None))
-                for i, content in enumerate(content_array)]
-    if input_type == 'FRAME' and json_content_array is None:
-        content_array = [encode_object_if_not_url(
-            content, input_type) for content in content_array]
+        return [content if is_url(content) else (content
+            if (json_content_array is not None and json_content_array[i] is not None)
+            else (encode_base64(content) if check_file_mime_type(content, input_type) else None))
+            for i, content in enumerate(content_array)]
+    if input_type in ('VIDEO', 'FRAME') and json_content_array is None:
+        if input_type == 'FRAME':
+            warnings.warn("FRAME input type is deprecated. Please use VIDEO instead")
```

Comment:
Same here, is it useful to raise such a warning each time that he push new assets ?